packages/framework-integration-tests/integration/cli/cli.project.integration.ts

```diff
@@ -297,8 +297,23 @@ describe('Project', () => {
         }).timeout(TEST_TIMEOUT)
 
         it('compiles', async () => {
+          // To compile the project with the current version we need to replace all Booster dependencies
+          // by the versions under development.
+          for (const packageName of ['framework-core', 'framework-types', 'framework-provider-aws']) {
+            await exec(`npm install ../../../${packageName}`, { cwd: projectPath(projectName) })
+          }
+
+          for (const packageName of ['cli', 'framework-provider-aws-infrastructure']) {
+            await exec(`npm install -D ../../../${packageName}`, { cwd: projectPath(projectName) })
+          }
+
+          await exec('npm install', { cwd: projectPath(projectName) })
+          // For some reason, the project doesn't compile if we don't re-compile the modules.
+          // Maybe rewriting the dependencies messes up lerna's tricks ¯\_(ツ)_/¯
+          await exec('lerna clean --yes && lerna bootstrap && lerna run clean && lerna run compile')
+
```

Comment:
This is crazy slow. For some reason, I was forced to rebuild all modules to make the project compile, but with this change, we're using the code under development instead of the libraries installed from npm. I don't know if there are other places in the code that would need to use this trick. It's very likely that we could optimize this in the future, but for this PR, this is more than enough...